gk/fib.c

```diff
@@ -20,6 +20,7 @@
 #include <arpa/inet.h>
 #include <linux/rtnetlink.h>
 
+#include "gatekeeper_config.h"
```

Comment:
This line of the GK block should be on a patch on its own since it's not related to the goal of this patch. In the new patch, comment why this header is being added.

Reply:
I think it is related to this patch because it is a side effect of removing `include/gatekeeper_acl.h` from `include/gatekeeper_lls.h`.

This is how `gk/fib.c` was using `include/gatekeeper_config.h`:

```
include/gatekeeper_config.h
^
|
include/gatekeeper_acl.h
^
|
include/gatekeeper_lls.h
^
|
gk/fib.c
```

Reply:
Well taken. Thank you for the detailed explanation. Issue solved.